cekit/generator/base.py

```diff
@@ -723,21 +724,25 @@ def add_module(self, module: Module):
                 )
             )
 
-        default_version = parse_version(self._defaults.get(module.name))
-        current_version = parse_version(version)
-
-        if isinstance(current_version, LegacyVersion):
-            LOGGER.warning(
-                (
-                    "Module's '{}' version '{}' does not follow PEP 440 versioning scheme "
-                    "(https://www.python.org/dev/peps/pep-0440), "
-                    "we suggest follow this versioning scheme in modules"
-                ).format(module.name, version)
+        default_version: Version = parse_version(self._defaults.get(module.name))
+        try:
+            current_version: Version = parse_version(version)
+            # https://github.com/pypa/packaging/issues/631#issuecomment-1343508993
+            if not hasattr(current_version, "major"):
```

Comment:
This will only happen under Python 3.6 if packaging is < 22 and returns a LegacyVersion. At this point we force it into the invalid version path to handle LegacyVersions.

Reply:
We should drop support for Python 3.6 since it is EOL.

Reply:
Dropping Python 3.6 could be considered but I think that should be handled separately.